docs/config.md

```diff
@@ -0,0 +1,547 @@
+    - org.wso2.bfsi.consent.management.common-1.0.0.jar
+    - org.wso2.bfsi.consent.management.dao-1.0.0.jar
+    - org.wso2.bfsi.consent.management.service-1.0.0.jar
+    - org.wso2.bfsi.consent.management.extensions-1.0.0.jar
+    - org.wso2.bfsi.identity.extensions-1.0.0.jar
+    - classmate-1.5.1.jar
+2. Copy the following artifacts to <IS_HOME>/repository/components/libs folder.
+    - commons-beanutils-1.9.4.jar
+    - validation-api-2.0.1.Final.jar
+    - hibernate-validator-6.0.20.Final.jar
+3. Copy the following artifacts to <IS_HOME>/repository/deployment/server/webapps folder.
+    - api#bfsi#consent.war
+    - bfsi#authenticationendpoint.war
+4. Copy the following configuration file to <IS_HOME>/repository/conf folder.
+    - bfsi-consent-management.xml
+
+### Setting up a new database
+
+1. Create a new database with the name - `bfsi_consentdb`.
+2. Create database tables by running the relevant the db script inside from the `dbscripts` folder.
+   
+   | DBMS Type                 | DB Script                                                            |
+   |---------------------------|----------------------------------------------------------------------|
+   | MySQL 8.0                 | <a href="../resources/mysql.sql" download> mysql.sql  </a>           |
+   | Oracle 19c                | <a href="../resources/oracle.sql" download> oracle.sql  </a>         |
+   | Microsoft SQL Server 2017 | <a href="../resources/mssql.sql" download> mssql.sql  </a>           |
+   | PostgreSQL 13             | <a href="../resources/postgresql.sql" download> postgresql.sql  </a> |
+
+3. According to your DBMS, place the compatible JDBC drivers in the following directories:
+ 
+       - `<IS_HOME>/repository/components/lib` 
+
+   !!! tip 
+   The supported JDBC driver versions are as follows:
+        
+       | DBMS version | JDBC driver version |
+       |--------------|---------------------|
+       | MySQL 8.0 | `mysql-connector-java-5.1.44.jar` |
+       | Oracle 19c | `ojdbc10.jar` |
+       | Microsoft SQL Server 2017 | `sqljdbc41.jar` |
+       | PostgreSQL 13 | `postgresql-42.2.17.jar` |
+
+### Configuring BFSI Consent Management Connector
+
+1. Add the root and issuer of the client certificate to the client trustore.
+``` bash
+keytool -import -alias <alias_name> -file <localtion_to-the-root-certificate> -keystore <IS_HOME>/repository/resources/security/client-truststore.jks -storepass wso2carbon
+```
+``` bash
+keytool -import -alias <alias_name> -file <localtion_to-the-issuer-certificate> -keystore <IS_HOME>/repository/resources/security/client-truststore.jks -storepass wso2carbon
+```
+
+2. Add the following configurations to the deployment toml file
+
+- Add the following data source configuration
+
+``` toml tab="MySQL"
+[[datasource]]
+id="WSO2BFSI_DB"
+url = "jdbc:mysql://localhost:3306/bfsi_consentdb?autoReconnect=true&amp;useSSL=false"
+username = "root"
+password = "root"
```

Comment:
```suggestion
url = "jdbc:mysql://<hostname:port>/bfsi_consentdb?autoReconnect=true&amp;useSSL=false"
username = "<username>"
password = "<password>"
```

Reply:
Fixed in https://github.com/wso2-extensions/bfsi-consent-management/pull/11/commits/d9302fc7f954430ce59b548ec2243eb849ba3330